src/zimscraperlib/imaging.py

```diff
@@ -68,15 +68,32 @@ def resize_image(
         else:
             resized = resizeimage.resize(method, image, [width, height])
 
+    # remove alpha layer if not supported and added during resizing
+    alpha_not_supported = ["JPEG", "BMP", "EPS", "PCX"]
+    if resized.mode == "RGBA" and image_format in alpha_not_supported:
+        resized = resized.convert(image_mode)
     # save the image
-    if resized.mode == "RGBA" and image_format == "JPEG":
-        resized = resized.convert("RGB")
     kwargs = {"JPEG": {"quality": 100}, "PNG": {}}
     resized.save(
-        str(to) if to is not None else fpath, image_format, **kwargs.get(image_format)
+        str(to) if to is not None else fpath,
+        image_format,
+        **kwargs.get(image_format, {}),
     )
 
 
+def change_image_format(src, dst, target_format, colorspace=None):
```

Comment:
other func is called `resize_image` so it makes more sense to use `convert_image`, right?

Reply:
Done